doc/man7/fips_module.pod

```diff
@@ -341,7 +345,7 @@ base providers. The other library context will just use the default provider.
     /* As an example get some digests */
 
     /* Get a FIPS validated digest */
-    fipssha256 = EVP_MD_fetch(fips_libctx, "SHA2-256", NULL);
+    fipssha256 = EVP_MD_fetch(fips_libctx, "SHA2-256", "fips=yes");
```

Comment:
I think it would be better in this example to use `EVP_default_properties_enable_fips` or `EVP_set_default_properties` to set "fips=yes" for the whole `fips_libctx` by default. This is the "safest" way to ensure that you always use FIPS validated algorithms - rather than setting the fetch specific properties which you are liable to forget at any specific call site (or if you call OpenSSL APIs that do a fetch internally).